app/jobs/submit_scan_request_job.rb

```diff
@@ -0,0 +1,44 @@
+class SubmitScanRequestJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(request)
+    response = client.get('func_request_webservice_new.make_request', request_params(request)) do |req|
+
+    end
+
+    r = SymphonyResponse.new(JSON.parse(response.body)['request_response'])
+
+    request.symphony_response = r
+    request.save
+  end
+
+  def request_params(request)
+    patron_from_request(request).merge(items_from_request(request)).merge(
+      req_type: 'SCAN'
+    ).reject { |k, v| v.blank? }
+  end
+
+  def patron_from_request(request)
```

Comment:
Assignment Branch Condition size for patron_from_request is too high. [16.28/15]